CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+ 
+## v0.2.0
+ 
+### :bug: Breaking
```

Comment:
They were dropped on the backend, so this is not a breaking change, right? :thinking: